src/test/java/algorithms/leetcode/easy/MaxWordsInSentenceTest.java

```diff
@@ -0,0 +1,15 @@
+package algorithms.leetcode.easy;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MaxWordsInSentenceTest {
+
+    @Test
+    void testMaxWordsInSentence(){
+        int expectedWords = 6;
+        int actualWords = MaxWordsInSentence.mostWordsFound(new String[]{"alice and bob love leetcode", "i think so too", "this is great thanks very much"});
```

Comment:
Suggestion: if the line is going very long then break that line into multiple lines.